web-api/src/business/useCases/user/verifyUserPendingEmailInteractor.ts

```diff
@@ -24,7 +24,7 @@ export const userTokenHasExpired = (
     calculateDifferenceInHours(
       createISODateString(),
       tokenExpirationTimestamp,
-    ) > 1
+    ) > TOKEN_EXPIRATION_TIME_HOURS
```

Comment:
It is supposed to be 24 hours. I had this wrong originally (because the ticket said 1 hour), then updated it correctly, and then refactored to use a DateHelper function and got it wrong again 🤦 

We have good tests for expired and non-expired tokens. Another test to prevent the above would be something like, "Make sure the constant is correct," e.g., "Make sure a token isn't invalid after an hour, or maybe two hours, etc." Is that worth it? Since it is testing the specific implementation so rigidly, I think it isn't. But I can add one if anyone disagrees.